app/src/utils/data/ipfs.ts

```diff
@@ -14,7 +14,8 @@ function assertIPFSPointer(logoPtr: MetaPtr | undefined) {
     logoPtr = decodeMetadataId(logoPtr);
   }
   const protocol = BigNumber.from(logoPtr.protocol).toString();
-  if (protocol !== '1') throw new Error(`assertIPFSPointer: Expected protocol ID of 1, found ${protocol}`);
+  if (!['0', '1'].includes(protocol))
+    throw new Error(`assertIPFSPointer: Expected protocol ID of 0 or 1, found ${protocol}`);
```

Comment:
Pushed f1fd499 to revert this because this file shouldn't be aware of protocol ID 0